modules/dxp/apps/portal-workflow/portal-workflow-kaleo-metrics-integration/src/main/java/com/liferay/portal/workflow/kaleo/metrics/integration/internal/helper/IndexerHelper.java

```diff
@@ -124,6 +128,38 @@ public AddNodeRequest createAddNodeRequest(
 		).build();
 	}
 
+	public AddProcessRequest createAddProcessRequest(
+		KaleoDefinition kaleoDefinition) {
+
+		AddProcessRequest.Builder builder = new AddProcessRequest.Builder();
+
+		String defaultLanguageId = LocalizationUtil.getDefaultLanguageId(
+			kaleoDefinition.getTitle());
+
+		return builder.active(
+			kaleoDefinition.isActive()
+		).companyId(
+			kaleoDefinition.getCompanyId()
+		).createDate(
+			kaleoDefinition.getCreateDate()
+		).description(
+			kaleoDefinition.getDescription()
+		).modifiedDate(
+			kaleoDefinition.getModifiedDate()
+		).name(
+			kaleoDefinition.getName()
+		).processId(
+			kaleoDefinition.getKaleoDefinitionId()
+		).title(
+			kaleoDefinition.getTitle(defaultLanguageId)
```

Comment:
@seltong, set the content of defaultLanguageId variable inline, please.

Thanks!